### PR TITLE
New Nuke Icons, new Medical HUD sprites

### PR DESCRIPTION
**Pull request by Repth1**
http://i.imgur.com/gwdXDYo.png

Nuke icon has perspective now.

Medical HUDs:

> No bar at full health
> Smaller icons to not interfere with speech bubbles
> Thinner bar so it doesn't block the head as much removing buffer zone on top.
> Faster flashing state when between -50 to -99 health

### CODE REVIEW

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -117,8 +117,10 @@ proc/RoundHealth(health)
 			return "health10"
 		if(1 to 5)
 			return "health1"
-		if(-99 to 0)
+		if(-49 to 0)
 			return "health0"
+		if(-99 to 50)
```
Comment (unid15) [inline code/__HELPERS/mobs.dm]: I think you meant (-99 to -50)

File: code/_onclick/hud/pai.dm [hunk 2]
```diff
@@ -105,14 +105,16 @@
...
 			return "health25"
-		if(5 to 15)
+		if(5 to 18)
 			return "health10"
 		if(1 to 5)
 			return "health1"
-		if(-99 to 0)
+		if(-49 to 0)
 			return "health0"
+		if(-99 to 50)
```
Comment (unid15) [inline code/_onclick/hud/pai.dm]: same here

**Comment by Repth1** [comment]
@unid15 fug, fixed. Also don't ask me why there are 3 copied of the same proc, I don't even work here.

**Comment by dylanstrategie** [comment]
Complaints : 

- Keep the medical cross red like it was before. It must shine with the blood of its victims
- No outlines on the nuclear bomb please
- The anchor bolts look weird. They might gain to be thicker and not angled. Those are anchoring bolts, not wheelstoppers

**Comment by Repth1** [comment]
@dylanstrategie 

* Medical has been moving towards mostly green instead of red, so I thought it was appropriate.
* Was just basing it off other objects in the stationobjs file, it kinda looks weird without them.
* Fair dos

**Comment by dylanstrategie** [comment]
Do note that by outlines, I mean the weird neon green outline on the bolted nuke

**Comment by Repth1** [comment]
@dylanstrategie Oh, that's the sprite used (or not used the code is strange) when the nuke is about to explode. It's more of an easter egg since it'll never actually be seen anyway.

**Comment by dylanstrategie** [comment]
Actually, the ghosts almost always spectate the nuke when it is exploding

**Comment by Probe1** [comment]
:+1:

**Pull request merged**